### PR TITLE
Headless token refresh

### PR DESCRIPTION
**Pull request by KalleStruik**
Use `reddit_session` cookie to refresh the accessToken on headless bots. Currently still testing.  
This PR changes the bot.js script to take a semicolon (`;`) separated list of cookies instead of a comma (`,`) separated list of accessTokens. The env variable is also changed from `ACCESS_TOKEN` to `REDDIT_SESSION` to reflect this  

### How to get `reddit_session` cookie
1. Go to r/place
2. Open dev tools and go to the network tab
3. Refresh the page
4. Click on the first request to reddit.com/r/place (See image)
Screenshot_20220403_165251
5. Go to the tab called `Cookies`
6. Copy the value of the `reddit_session` cookie

### CODE REVIEW

**Comment by FloGo3308** [comment]
dit is erg hard nodig!!!

**Comment by SpiderDead** [comment]
Dit lijkt bij mij te werken. Zit zelf nog te wachten op een token refresh of dat goed werkt.

**Comment by nuc1e4r5n4k3** [comment]
Do we know if the reddit session cookie remains static or of it can update? If the latter we should capture the Set-Cookie header on the response and update `redditSessionCookies`.

**Comment by KalleStruik** [comment]
> Do we know whether the reddit session cookie can change? If so, we should capture the Set-Cookie header of the response and update `redditSessionCookies` with it.

I am pretty sure it is static (or at least valid for a long time), but I don't think it hurts to just do this anyway.

**Comment by KalleStruik** [comment]
Token refresh just happened on my end btw and it seems to still be placing pixels after it.

**Comment by Inktvisje** [comment]
Initial token request works on my end as well, waiting for the refresh now. so far so good!

File: bot.js [hunk 1]
```diff
@@ -182,6 +226,7 @@ async function attemptPlace(accessToken) {
     const data = await res.json();
     try {
         if (data.errors) {
+            console.log(data.errors)
```
Comment (Inktvisje) [inline bot.js:229]: Forgot to clean this up?

File: bot.js [hunk 2]
```diff
@@ -8,15 +8,23 @@ console.log(`PlaceNL headless client V${VERSION_NUMBER}`);
```
Comment (NoahvdAa) [inline bot.js:8]: ^ VERSION_NUMBER to 3

File: bot.js (hunk 1 above)
Comment (KalleStruik) [reply bot.js]: Yup removed it now.

**Comment by KalleStruik** [comment]
> Do we know whether the reddit session cookie can change? If so, we should capture the Set-Cookie header of the response and update `redditSessionCookies` with it.

Seems like this won't be easy to do after all, since the response to our request to /r/place does not return a set-cookie header for this cookie.

**Comment by SpiderDead** [comment]
Refreshing works. Just auto refreshed 2 clients and placed a tile.

**Review by NoahvdAa — APPROVED** [review]

**Pull request merged**